.circleci/config.yml

```diff
@@ -80,6 +80,9 @@ jobs:
             docker exec -it pthd /bin/bash -c "$test1_cmd"
             export test2_cmd='export WORLD_SIZE=1 && py.test --dist=each --tx $WORLD_SIZE*popen//python=python3.7 tests -m distributed -vvv'
             docker exec -it pthd /bin/bash -c "$test2_cmd"
+            # Install ignite in docker
+            export install_cmd='pip install .'
+            docker exec -it pthd /bin/bash -c "$install_cmd"
```

Comment:
It would be better to install ignite at `install_dependencies: &install_dependencies` step ?